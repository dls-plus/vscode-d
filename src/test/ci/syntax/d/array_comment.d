auto a = ["a", // comment
    "b"];