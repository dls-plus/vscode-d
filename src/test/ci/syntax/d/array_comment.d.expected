[{"start":0,"end":4,"scope":"storage.type.basic-type.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"variable.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":8,"scope":"keyword.operator.assign.d"},{"start":8,"end":9,"scope":"source.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":11,"scope":"string.double-quoted-string.d"},{"start":11,"end":12,"scope":"string.character.d"},{"start":12,"end":13,"scope":"string.double-quoted-string.d"},{"start":13,"end":14,"scope":"keyword.operator.comma.d"},{"start":14,"end":15,"scope":"source.d"},{"start":15,"end":25,"scope":"comment.line.d"}]
[{"start":0,"end":4,"scope":"source.d"},{"start":4,"end":5,"scope":"string.double-quoted-string.d"},{"start":5,"end":6,"scope":"string.character.d"},{"start":6,"end":7,"scope":"string.double-quoted-string.d"},{"start":7,"end":8,"scope":"source.d"},{"start":8,"end":10,"scope":"meta.statement.end.d"}]